meerkat_consul/export.py

```diff
@@ -291,15 +413,15 @@ def has_data_element_with_code(dhis2_code):
 
     @staticmethod
     def get_and_cache_value(dhis2_resource, dhis2_code):
-        cache = Dhis2CodesToIdsCache.caches[dhis2_resource]
+        cache = Dhis2CodesToIdsCache.caches.get(dhis2_resource, {})
         if not cache.get(dhis2_code):
             logger.info("{} with code {} not found in cache.".format(dhis2_resource, dhis2_code))
             rv = get("{url}/{resource_path}?filter=code:eq:{code}".format(
                 url=dhis2_api_url,
                 resource_path=dhis2_resource,
                 code=dhis2_code),
                 headers=dhis2_headers)
-            dhis2_objects = rv.json().get(dhis2_resource)
+            dhis2_objects = rv.json().get(dhis2_resource, [])
```

Comment:
That's better but we should have a proper error handling here.
if `rv.json().get(dhis2_resource)` is None it means we posted a bad request to dhis2.